jass/lib/CompileInfo.zn

```diff
@@ -2,15 +2,15 @@ library CompileInfo requires Constants, ArgsUtils {
 
   public struct CompileInfo {
 
-    static constant string  compiledBy = "<%= @compiled_by.gsub "\\", "/" %>";
+    static constant string  compiledBy = "";//"<%= @compiled_by.gsub "\\", "/" %>";
 
-    static constant integer year       = <%= Time.now.strftime("%Y").to_i %>;
-    static constant integer dayOfYear  = <%= Time.now.strftime("%j").to_i %>;
-    static constant integer month      = <%= Time.now.strftime("%m").to_i %>;
-    static constant integer day        = <%= Time.now.strftime("%d").to_i %>;
-    static constant integer hours      = <%= Time.now.strftime("%H").to_i %>;
-    static constant integer minutes    = <%= Time.now.strftime("%M").to_i %>;
-    static constant integer seconds    = <%= Time.now.strftime("%S").to_i %>;
+    static constant integer year       = 1;//<%= Time.now.strftime("%Y").to_i %>;
+    static constant integer dayOfYear  = 1;//<%= Time.now.strftime("%j").to_i %>;
+    static constant integer month      = 1;//<%= Time.now.strftime("%m").to_i %>;
+    static constant integer day        = 1;//<%= Time.now.strftime("%d").to_i %>;
+    static constant integer hours      = 1;//<%= Time.now.strftime("%H").to_i %>;
+    static constant integer minutes    = 1;//<%= Time.now.strftime("%M").to_i %>;
+    static constant integer seconds    = 1;//<%= Time.now.strftime("%S").to_i %>;
 
     static method operator dateTime () -> string {
       return I2S(year) + "-" + I2S(month) + "-" + I2S(day) + " " +
```

Comment:
My feeling is that we can just remove this file. I'm not sure its actually doing anything important.